common/src/chain/config.rs

```diff
@@ -40,6 +40,10 @@ pub struct ChainConfig {
 }
 
 impl ChainConfig {
+    pub fn chain_type(&self) -> ChainType {
```

Comment:
should this return a reference? All other getters of `ChainConfig` seem to return a reference

Reply:
Yes and no. Generally, yes, it should be. But since this is just an empty enum with no contents other than the numeric arms, it supports the copy trait, and is just an integer, so it doesn't really matter.

Reply:
i removed this, since getting the pow config is already a method under ChainConfig. I didn't expose the chain_type anymore.